app/assets/javascripts/discourse/app/widgets/post-menu.js

```diff
@@ -949,3 +951,35 @@ export default createWidget("post-menu", {
     }
   },
 });
+
+registerWidgetShim(
+  "glimmer-post-menu",
+  "section.post-menu-area.glimmer-post-menu-widget-shim.clearfix",
+  hbs`
+    <Post::Menu
+      @filteredRepliesView={{@data.filteredRepliesView}}
+      @model={{@data.model}}
+      @repliesShown={{@data.repliesShown}}
+      @transformedPost={{@data.transformedPost}}
```

Comment:
`transformedPost` is something which was introduced for widgets. So ideally, we would avoid using it in any new code. Can we get access to the actual post model, and use that? @tyb-talks made a similar change for the glimmer-ified topic map [here](https://github.com/discourse/discourse/commit/edbd44e737753eb4e9bd91c7da1836954e0627db)

Reply:
I've removed `trasnformedPost` from the new components in [fe7acac](https://github.com/discourse/discourse/pull/28670/commits/fe7acac052aa7d3f66350076e1732b6df00eb6a2)